src/main/java/duke/Event.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Event extends Task {
+    protected String at;
+    protected LocalDate parsedAt;
```

Comment:
maybe you could consider a different variable name like `date `since maybe that could fit more with the data type being LocalDate?